frontend/src/pages/Community.vue

```diff
@@ -0,0 +1,203 @@
+            creationDate: data.date,
+            memo: data.memo,
+            amount: data.amount,
+          },
+        })
+        .then((result) => {
+          // console.log('result', result.data)
+          this.toastSuccess(result.data)
+          this.updateListContributions({
+            currentPage: this.currentPage,
+            pageSize: this.pageSize,
+          })
+          this.verifyLogin()
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    updateListAllContributions(pagination) {
+      this.$apollo
+        .query({
+          fetchPolicy: 'no-cache',
+          query: listAllContributions,
+          variables: {
+            currentPage: pagination.currentPage,
+            pageSize: pagination.pageSize,
+          },
+        })
+        .then((result) => {
+          // console.log('result', result.data)
+          const {
+            data: { listAllContributions },
+          } = result
+          this.contributionCountAll = listAllContributions.contributionCount
+          this.itemsAll = listAllContributions.contributionList
+          // this.toastSuccess(result.data)
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    updateListContributions(pagination) {
+      this.$apollo
+        .query({
+          fetchPolicy: 'no-cache',
+          query: listContributions,
+          variables: {
+            currentPage: pagination.currentPage,
+            pageSize: pagination.pageSize,
+          },
+        })
+        .then((result) => {
+          // console.log('result', result.data)
+          const {
+            data: { listContributions },
+          } = result
+          this.contributionCount = listContributions.contributionCount
+          this.items = listContributions.contributionList
+          // this.toastSuccess(result.data)
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    verifyLogin() {
+      this.$apollo
+        .query({
+          query: verifyLogin,
+          fetchPolicy: 'network-only',
+        })
+        .then((result) => {
+          const {
+            data: { verifyLogin },
+          } = result
+          this.$store.dispatch('login', verifyLogin)
+        })
+        .catch(() => {
+          this.$emit('logout')
+        })
+    },
+    updateContributionForm(item) {
+      this.form.id = item.id
+      this.form.date = item.createdAt
+      this.form.memo = item.memo
+      this.form.amount = item.amount
+      this.tabIndex = 0
+    },
+    updateTransactions(pagination) {
+      this.$emit('update-transactions', pagination)
+    },
+  },
+  created() {
+    this.updateListContributions({
+      currentPage: this.currentPage,
+      pageSize: this.pageSize,
+    })
+    this.updateListAllContributions({
+      currentPage: this.currentPage,
+      pageSize: this.pageSize,
+    })
+    this.updateTransactions(0)
```

Comment:
Why is this needed?

Reply:
Yes, we need that at this point, unfortunately, because otherwise the GDD amount is not displayed when I call up the page alone or reload it.